process/peer/process.go

```diff
@@ -866,3 +885,30 @@ func (vs *validatorStatistics) decreaseAll(shardId uint32, missedRounds uint64,
 
 	return nil
 }
+
+// Process - processes a validatorInfo and updates fields
+func (vsp *validatorStatistics) Process(vid *state.ValidatorInfo) error {
+	log.Trace("ValidatorInfoData", "pk", vid.GetPublicKey(), "rating", vid.GetRating(), "tempRating", vid.GetTempRating())
+
+	pa, err := vsp.GetPeerAccount(vid.GetPublicKey())
+	if err != nil {
+		return err
+	}
+
+	err = pa.SetRatingWithJournal(vid.GetRating())
+	if err != nil {
+		return err
+	}
+
+	err = pa.SetTempRatingWithJournal(vid.GetTempRating())
+	if err != nil {
+		return err
+	}
+
+	err = pa.SetRatingWithJournal(vid.GetTempRating())
```

Comment:
why setting rating twice ? once with getRating and once with GetTempRating ?

Reply:
I wanted to "journalize" both the rating and tempRating. The peer processor in shards can just set the rating and nothing else. Changed